geode-core/src/main/java/org/apache/geode/internal/cache/TombstoneService.java

```diff
@@ -423,8 +425,8 @@ protected void updateStatistics() {
     }
 
     @Override
-    protected boolean hasExpired(long msTillHeadTombstoneExpires) {
-      return msTillHeadTombstoneExpires <= 0;
+    protected boolean hasExpired(long msUntilTombstoneExpires) {
+      return msUntilTombstoneExpires <= 0;
```

Comment:
We may have to add the large clock jump verification here too...I am assuming clients are also doing Tombstone GC similarly....Nice to add a test on the client side to verify it.

Reply:
I have added that change.